.travis.yml

```diff
@@ -1,8 +1,9 @@
+dist: xenial
 sudo: required
 language: java
 jdk:
-- oraclejdk8
-- oraclejdk11
+- openjdk8
+- openjdk11
```

Comment:
Should this be in another PR?

Reply:
could be but it fixes the current broken build

Reply:
Ok